specification/portal/resource-manager/Microsoft.Portal/preview/2019-01-01-preview/tenantConfiguration.json

```diff
@@ -0,0 +1,215 @@
+          },
+          {
+            "$ref": "portal.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "tenantConfiguration",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Configuration"
+            },
+            "description": "The parameters required to create or update tenant configuration."
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created. New configuration was created",
+            "schema": {
+              "$ref": "#/definitions/Configuration"
+            }
+          },
+          "200": {
+            "description": "OK. Existing configuration updated.",
+            "schema": {
+              "$ref": "#/definitions/Configuration"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "portal.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create or update Tenant configuration": {
+            "$ref": "examples/TenantConfiguration/CreateOrUpdateTenantConfiguration.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "tenantConfiguration"
+        ],
+        "operationId": "TenantConfiguration_Delete",
+        "description": "Delete the tenant configuration.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ConfigurationNameParameter"
+          },
+          {
+            "$ref": "portal.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. Configuration was deleted."
+          },
+          "204": {
+            "description": "No content. Configuration was not found."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "portal.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Tenant configuration": {
+            "$ref": "examples/TenantConfiguration/DeleteTenantConfiguration.json"
+          }
+        }
+      }
+    }
+  },
+  "parameters": {
+    "ConfigurationNameParameter": {
+      "name": "configurationName",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The configuration name. Value must be 'default'",
+      "enum": [
+        "default"
+      ],
+      "x-ms-enum": {
+        "name": "configurationName",
+        "modelAsString": true
+      },
+      "x-ms-parameter-location": "method"
+    }
+  },
+  "definitions": {
+    "Configuration": {
+      "description": "Tenant configuration.",
+      "x-ms-azure-resource": true,
+      "properties": {
+        "id": {
```

Comment:
You should use the common-types to get id, name and type fields.
Example:
https://github.com/Azure/azure-rest-api-specs/blob/95869c4461d72cc2c2bba489e23b9ccfcea75169/specification/azure-kusto/resource-manager/Microsoft.Kusto/stable/2019-05-15/kusto.json#L1983